proto/build.rs

```diff
@@ -251,6 +258,26 @@ static FIELD_ATTRIBUTES: &[(&str, &str)] = &[
     (".penumbra.transaction.OutputPlan.esk", AS_HEX_FOR_BYTES),
     // TODO: replace if we use UTF-8 memos
     (".penumbra.transaction.OutputPlan.memo", AS_HEX_FOR_BYTES),
+    // TODO: these aren't right yet, the Swap/SwapClaim proto messages don't have the correct fields
+    (
+        ".penumbra.transaction.SwapPlan.randomizer",
+        AS_HEX_FOR_BYTES,
+    ),
+    (
+        ".penumbra.transaction.SwapPlan.value_blinding",
+        AS_HEX_FOR_BYTES,
+    ),
+    (
+        ".penumbra.transaction.SwapClaimPlan.note_blinding",
+        AS_HEX_FOR_BYTES,
+    ),
+    (
+        ".penumbra.transaction.SwapClaimPlan.value_blinding",
+        AS_HEX_FOR_BYTES,
+    ),
+    (".penumbra.transaction.SwapClaimPlan.esk", AS_HEX_FOR_BYTES),
+    // TODO: replace if we use UTF-8 memos
+    (".penumbra.transaction.SwapClaimPlan.memo", AS_HEX_FOR_BYTES),
```

Comment:
Wait, why does a `SwapClaim` have a memo at all?

Reply:
this is outdated/copied from `Output` and needs removal, this PR is still a draft